include/config/config_game.h

```diff
@@ -26,6 +29,9 @@
 // Enables the beep sound at low health (vanilla behavior)
 #define ENABLE_LOW_HEALTH_BEEP true
 
+// Clamps the jumping velocity (vanilla behavior)
```

Comment:
I think the comment should mention disabling this is giving jumps the MM behavior (except the animation I guess)